packages/editor/src/components/block-list/block.js

```diff
@@ -357,6 +366,12 @@ export class BlockListBlock extends Component {
 		}
 	}
 
+	handleAltF10KeyPress() {
```

Comment:
The naming here feels awfully like an implementation detail of the current keybind, not the intended behavior.

Maybe something like `forceFocusedContextualToolbar`